routes/web3relay.js

```diff
@@ -31,6 +31,23 @@ else
 var newBlocks = web3.eth.filter("latest");
 var newTxs = web3.eth.filter("pending");
 
+exports.totalSupply = function (req, res) {
+  web3.currentProvider.sendAsync({
+    jsonrpc: "2.0",
+    method: "eth_totalSupply",
+    params: ['0x0'],
```

Comment:
Is this requesting the total supply of block #0 every time? No params should default to 'latest', like on the console here:
![screenshot from 2018-06-28 12-02-33](https://user-images.githubusercontent.com/1194128/42049305-2fef44da-7acb-11e8-9507-5657d5a60c0f.png)

Reply:
Oh, and the result is in `wei`, not `GO`, just like `getBalance`.